docling-server/src/main.py

```
@@ -0,0 +1,115 @@
+        debug_dir.mkdir(exist_ok=True)
+        timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
+        debug_path = debug_dir / f"{doc_id}_{timestamp}.json"
+        with open(debug_path, "w") as f:
+            json.dump(result, f, indent=2, default=str)
+        print(f"[DEBUG] Result saved to: {debug_path}")
+
+        return JSONResponse(content=result)
+
+    except HTTPException:
+        raise
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)